### PR TITLE
done hw

### PR DESCRIPTION
**Pull request by mnovosad1095**

### CODE REVIEW

**Comment by pavlo-berezin** [comment]
@mnovosad1095 @MANOVOS Please help!
image

**Comment by mnovosad1095** [comment]
@pavlo-berezin, npm install @material-ui/core ?

**Comment by mnovosad1095** [comment]
@pavlo-berezin, hw2 done

File: src/App.css [hunk 1]
```diff
@@ -36,3 +36,15 @@
...
 }
+
+body {
+  background: linear-gradient(to right, #4ac29a, #bdfff3); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+}
+
+
+footer {
+  position:absolute;
+left:0;
```
Comment (pavlo-berezin) [inline src/App.css]: fix indentation.

File: src/App.js [hunk 2]
```diff
@@ -1,26 +1,21 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import HomeGrid from './components/HomeGrid';
+import NavBar from './components/NavBar';
+import { makeStyles } from '@material-ui/core/styles';
```
Comment (pavlo-berezin) [inline src/App.js:5]: unused import

File: src/App.js [hunk 3]
```diff
@@ -1,26 +1,21 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return (<div className={classes.root}><center><NavBar/><HomeGrid /></center><StickyFooter /></div>);
```
Comment (pavlo-berezin) [inline src/App.js]: don't put so many elements in one line.

File: src/components/HomeGrid.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import CardActionArea from '@material-ui/core/CardActionArea'
+
+
+import ImageChords from '../img/chords.jpg'; // Import using relative path
+import ImageInters from '../img/intervals1.jpg';
+import ImageProgs from '../img/sheet-music.jpg';
+
+const card =  (theme) => ({
```
Comment (pavlo-berezin) [inline src/components/HomeGrid.js]: extra space.

File: src/components/HomeGrid.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+        
+    },
+    cardChord: Object.assign({}, card(theme), {"backgroundImage": `url(${ImageChords})`}),
+    cardProg: Object.assign({}, card(theme), {"backgroundImage": `url(${ImageInters})`}),
+    cardInters: Object.assign({}, card(theme), {"backgroundImage": `url(${ImageProgs})`}),
+}));
+
+function gridComp(className, xs, text) {
+    return (
+    <Grid item xs={xs}>
```
Comment (pavlo-berezin) [inline src/components/HomeGrid.js]: fix indentation.

File: src/components/footer.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+function Copyright() {
+  return (
+    <Typography variant="body2" color="textSecondary">
+      {'Copyright © '}
+      <Link color="inherit" href="#">
+        My Website
+      </Link>{' '}
+      {new Date().getFullYear()}
+      {'.'}
```
Comment (pavlo-berezin) [inline src/components/footer.js:15]: You can add this in css with `:after`

**Pull request merged**